Extensions/ParticleSystem/ExtensionSubDeclaration3.cpp

```diff
@@ -142,6 +142,7 @@ void ExtensionSubDeclaration3(gd::ObjectMetadata& obj) {
                 "CppPlatform/Extensions/particleSystemicon24.png",
                 "CppPlatform/Extensions/particleSystemicon16.png")
       .AddParameter("object", _("Object"), "ParticleEmitter")
+      .SetParameterLongDescription(_("Valid name is the resource name used for an asset."))
```

Comment:
I think a clearer sentence could be: `Please indicate the name of the resource` and maybe `(you can find it in the Resources manager in the project manager)`